tests/integration/sharding_tests/helpers.py

```diff
@@ -107,3 +115,68 @@ def count_users(mongos_client: MongoClient) -> int:
     admin_db = mongos_client["admin"]
     users_collection = admin_db.system.users
     return users_collection.count_documents({})
+
+
+async def deploy_cluster_components(
```

Comment:
These helpers were already made in the TLS tests for sharding, but are quite useful for other tests so I moved them into the sharding test helpers  ⭐